src/commonMain/kotlin/com/github/quillraven/fleks/family.kt

```diff
@@ -233,15 +236,17 @@ data class Family(
     @PublishedApi
     internal fun onEntityCfgChanged(entity: Entity, compMask: BitArray) {
         val entityInFamily = compMask in this
-        if (entityInFamily && !entityBits[entity.id]) {
+        if (entityInFamily && privateEntities.hasNoValueAtIndex(entity.id)) { // !entityBits[entity.id]
```

Comment:
Extracting the `hasNoValueAtIndex` result would be good because in worst case this lookup is executed twice. We can reduce it to one. Not a huge improvement but still ;)